website/docs/patterns/description-list-patterns/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,113 @@
+## Typography
+
+### Heading
+
+Use `Body 200 Semibold` with `Foreground / Primary` for headings.
+
+![](/assets/patterns/description-list-patterns/description-list-heading.png)
+
+### Value
+
+#### Standard
+
+Use `Body 200 Regular` with `Foreground / Strong` for values. This style is used when the heading and value are of equal importance.
+
+![](/assets/patterns/description-list-patterns/description-list-value-standard.png)
+
+#### Callout
+
+This style is used to emphasize values over headings. It is often used for numerical content, like resources or money. Use `Display 300 / 400 Bold` (depending on content hierarchy) with `Foreground / Strong`. This should only be used for vertical orientation description lists.
+
+![](/assets/patterns/description-list-patterns/description-list-value-callout.png)
+
+!!! Dont
+
+Mix and match visual styles. Ensure visual consistency is maintained by choosing one style in a given space.
+
+![A card with five columns of headings and values. The last column using the callout style while the rest using the standard style.](/assets/patterns/description-list-patterns/description-list-dont-mix-match-styles.png)
+
+!!!
+
+### Alignment
+
+We recommend that content is left aligned, even if the values are related to other content on the page. This maintains the natural reading flow of the content.
+
+![A description list with three columns contained in a card sitting above a table. Each list items is aligned to the left while the values in the table in the last columnn are right aligned.](/assets/patterns/description-list-patterns/description-list-alignment-left.png)
+ 
+## Orientation
+
+A description list can be oriented either vertically or horizontally.
+
+### Vertical
+
+Use vertical orientation for designs similar to metacards or summary card experiences. With this orientation, the value in the description list can be styled either using the standard or the callout style.
+
+#### Spacing between the heading and value
+
+Use an 8px gap in between the heading and value. Components can be used in place of the value, but only one component per description list item is allowed.
+
+![Two rows of five columns with description list items. First row using the standard style while the second row using the callout style.](/assets/patterns/description-list-patterns/description-list-vertical-spacing-heading-value.png)
+
+In rare instances where the value slot requires additional rows for context, the gap is 4px. The content within the value slot can have multiple rows. Consider the visual hierarchy of the overall grouping during implementation.
```

Comment:
[suggestion] this more consistent with the other language that has been pivoted to say "use"

```suggestion
In rare instances where the value slot requires additional rows for context, use a 4px gap. The content within the value slot can have multiple rows. Consider the visual hierarchy of the overall grouping during implementation.
```